flashCards.rb

```diff
@@ -0,0 +1,123 @@
+require "pry"
+class Flashcard 
+	attr_accessor :front, :back
+	def initialize
+
+	@front = "front"
+    @back = "back"
+    @new_front = []
+    @new_back = []
+
+    @@default_cards = [@front, @back]
+    @@new_cards = [@new_front, @new_back]
```

Comment:
That might change how you have to handle creating/updating cards in your app, but ultimately I think that change would go towards simplifying your code.